local install not working

Hi, i am using ubuntu 20.04 lts, i have manually built opensta and it still saying opentimer not found. At the end it also throw exception that i can't understand, thanks

```
Using yosys verilog synthesis tool at: /usr/bin/yosys
Using graywolf placement tool at: /usr/bin/graywolf
RePlAce not found.  Use --with-replace=<DIR> (optional)
Using qrouter detail route tool at: /usr/bin/qrouter
Using Magic layout tool at: /usr/bin/magic
Using Netgen LVS tool at: /usr/bin/netgen
Using Vesta STA tool (internal)
OpenTimer not found.  Use --with-opentimer=<DIR> (optional)
Using OpenSTA static timing analysys tool at: /usr/local/bin/sta
----------------------------------------------------
All tools needed for complete flow have been found.
Run "make" followed by "sudo make install" to install qflow.

----------------------------------------------------
rm -f -f vlog2Spice.o vlog2Verilog.o vlog2Def.o vlog2Cel.o vlogFanout.o DEF2Verilog.o addspacers.o vesta.o spice2delay.o rc2dly.o blif2BSpice.o blif2Verilog.o blifFanout.o hash.o readliberty.o readlef.o readdef.o readverilog.o
rm -f -f vlog2Spice vlog2Verilog vlog2Def vlog2Cel vlogFanout DEF2Verilog addspacers vesta spice2delay rc2dly blif2BSpice blif2Verilog blifFanout
make: *** No rule to make target 'vlogtoverilog'.  Stop.
Traceback (most recent call last):
  File "./env.py", line 536, in <module>
    main()
  File "./env.py", line 533, in main
    commands[args[0]]()
  File "./env.py", line 407, in local_install
    installer.run()
  File "./env.py", line 393, in run
    install()
  File "./env.py", line 47, in __exit__
    raise exc_value
  File "./env.py", line 393, in run
    install()
  File "./env.py", line 377, in install
    f.write(tool.version_string)
  File "./env.py", line 47, in __exit__
    raise exc_value
  File "./env.py", line 368, in install
    subprocess.run([
  File "./env.py", line 47, in __exit__
    raise exc_value
  File "./env.py", line 368, in install
    subprocess.run([
  File "./env.py", line 47, in __exit__
    raise exc_value
  File "./env.py", line 368, in install
    subprocess.run([
  File "/usr/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['bash', '-c', '                                    set -e\n                                    source /home/peter/workspace/OpenLane/install/venv/bin/activate\n                                    # Note that vlogtoverilog is part of the qflow repository.\n./configure\ncd src\nmake clean\nmake -j$(nproc) vlogtoverilog\ncp vlogtoverilog $PREFIX/bin\n\n
       ']' returned non-zero exit status 2.
```

--- a/dependencies/tool_metadata.yml
+++ b/dependencies/tool_metadata.yml
@@ -66,8 +66,8 @@
     ./configure
     cd src
     make clean
-    make -j$(nproc) vlogtoverilog
-    cp vlogtoverilog $PREFIX/bin
+    make -j$(nproc) vlog2Verilog
+    cp vlog2Verilog $PREFIX/bin
 - name: yosys
   repo: https://github.com/YosysHQ/yosys
   commit: d061b0e41a2023b5e72794563b94d6a9b5ab41a1

--- a/dependencies/ubuntu-20.04/run_time.txt
+++ b/dependencies/ubuntu-20.04/run_time.txt
@@ -34,8 +34,8 @@ patch
 libpcre2-dev
 python3
 python3-pip
-python3.6-dev
-python3.6-tk
+python3-dev
+python3-tk
 libreadline-dev
 strace
 swig

